examples/cloudtrail/trail.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: cloudtrail.aws.upbound.io/v1beta1
+kind: Trail
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudtrail/v1beta1/trail
+  labels:
+    testing.upbound.io/example-name: foobar
+  name: foobar
+spec:
+  forProvider:
+    includeGlobalServiceEvents: false
+    region: us-west-1
+    s3BucketNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: foo
+    s3KeyPrefix: prefix
+
+---
+
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    crossplane.io/external-name: op-rgszzcuww 
```

would be good to use a random external-name in here too

```suggestion
    crossplane.io/external-name: ${Rand.RFC1123Subdomain}
```

---

As we discussed offline, we need that name in the BucketPolicy resource. So we can keep it hard-coded.